cmd/kube-scheduler/app/options/options.go

```diff
@@ -231,6 +233,11 @@ func (o *Options) ApplyTo(c *schedulerappconfig.Config) error {
 		}
 	}
 	o.Metrics.Apply()
+
+	if o.Deprecated != nil {
```

Comment:
Add a comment about why this is not part of ApplyDeprecated.

Presumably because this can't be configured via ComponentConfig.